Woof/Woof/View/SitterFlow/SitterWalkingsView/SitterWalkingsSectionView.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+
+/// Represents a section of walkings, with an expandable list of walkings.
+struct SitterWalkingsSectionView: View {
+    /// Control the expansion state of the walking list.
+    @Binding var isExpanded: Bool
```

Comment:
We don't need to expose this property, do we?
```suggestion
    @State private var isExpanded: Bool
```

Reply:
As we decide in the parent view which section was exposed or collapsed we need the binding. We show only 2 expanded sections - the current and the future section.
We can remove isExpanded from the initializer, but all the sections will be collapsed by default.

Reply:
We can do in this way:

```swift
    @State private var isExpanded: Bool
    init(
        isExpanded: Bool,
        walkings: [Walking],
        headerTitle: String)
    {
        self.isExpanded = isExpanded
        self.walkings = walkings
        self.headerTitle = headerTitle
    }
```

Reply:
Fixed